roma/romadom.php

```diff
@@ -2292,20 +2292,85 @@ public function createSchemaISOSCH( &$szISOSCH )
 
     public function createSchemaRNC( &$szRNC )
       {
-	$target="rnc%3Aapplication%3Arelaxng-compact/";
-	$this->callGarage($szRNC, $target);
+	$target="relaxng%3Aapplication%3Axml-relaxng/";
```

Comment:
and here: 
https://github.com/TEIC/Roma/commit/55a51a433983a0b9c9eaf59f98478b25cfc50290